### PR TITLE
constitution changes voted at NIAC2018

### PR DESCRIPTION
**Pull request by prjemian**
see https://github.com/nexusformat/NIAC/issues/71

### CODE REVIEW

File: content/NIAC.md [hunk 1]
```diff
@@ -117,13 +117,27 @@ executive officers:
...
+committee. However, all debates will be conducted publicly and 
+non-committee members are both allowed and encouraged to participate.  
+
 Topics voted on will require a simple majority unless specifically
 challenged by a committee member. In the event of a challenge, the issue
 will be presented to the NeXus mailing list inviting comments for a
 period of one month. At the end of this period, the challenge can be
 overruled by a two thirds majority.
 
+Email voting as used up to 2018 only for membership renewals.
```
Comment (prjemian) [inline content/NIAC.md:131]: This appears to describe history and is not a statement for the constitution.  This does not describe the email voting process.  We should add a statement that describes the email voting process.

Comment (prjemian) [reply content/NIAC.md:131]: Specifically, we should describe the process for membership renewals.

Comment (prjemian) [reply content/NIAC.md:131]: We'll publish as written and work on a revision for the final NIAC2020 close out session.

**Review by FreddieAkeroyd — APPROVED** [review]
Agree these are additions from NIAC2018 but we need to clarify some bits at NIAC 2020

File: content/NIAC.md (hunk 1 above)
Comment (benajamin) [reply content/NIAC.md:131]: I agree that this statement looks out of place. The constitution should only give the current rules and not comment on previous rules.

File: content/NIAC.md [hunk 2]
```diff
@@ -117,13 +117,27 @@ executive officers:
...
+
+For issues other than membership renewals, the following rule applies: 
+we vote by teleconference where the proposal can be discussed. At least 
+2/3 of the NIAC have to be present at the teleconference. Then normal 
+majority vote of the attendees applies. Teleconference decisions need to 
+be published on the NIAC mailing list. NIAC Members not attending the 
+teleconference have a one week period in which they can add their vote 
+to the tally. Those not responding in that week are counted as abstained.
+
```
Comment (FreddieAkeroyd) [inline content/NIAC.md:140]: This says we need 2/3 attendance for a simple majority vote, but then we send for email so it is not a simple majority vote?

**Review by benajamin — APPROVED** [review]
I think we could improve the wording to make it more clear

File: content/NIAC.md (hunk 2 above)
Comment (PeterC-DLS) [reply content/NIAC.md:140]: Could be clearer: it is always a simple majority decision only that when the vote is counted is delayed if there isn't a quorum.

File: content/NIAC.md (hunk 1 above)
Comment (prjemian) [reply content/NIAC.md:131]: Agree this should be more clear. Added the words as approved by NIAC2018.  We should propose changes to the close out session of NIAC2020.

**Pull request merged**